librz/core/cmd/cmd_analysis.c

```diff
@@ -8812,3 +8862,236 @@ RZ_IPI RzCmdStatus rz_analysis_class_vtable_lookup_handler(RzCore *core, int arg
 	ls_free(classes);
 	return RZ_CMD_STATUS_OK;
 }
+
+RZ_IPI RzCmdStatus rz_analyze_bytes_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
+	ut8 *buf;
+	int len;
+
+	if (!(buf = malloc(strlen(argv[1]) + 1))) {
+		return RZ_CMD_STATUS_ERROR;
+	}
+
+	len = rz_hex_str2bin(argv[1], buf);
+	if (len <= 0) {
+		free(buf);
+		return RZ_CMD_STATUS_ERROR;
+	}
+
+	switch (state->mode) {
+	case RZ_OUTPUT_MODE_JSON:
+		core_analysis_bytes_json(core, buf, len, 0, state->d.pj);
+		break;
+	case RZ_OUTPUT_MODE_STANDARD:
+		core_analysis_bytes_standard(core, buf, len, 0);
+		break;
+	default:
+		rz_warn_if_reached();
+	}
+
+	free(buf);
+	return RZ_CMD_STATUS_OK;
+}
+
+RZ_IPI RzCmdStatus rz_analyze_n_bytes_handler(RzCore *core, int argc, const char **argv, RzCmdStateOutput *state) {
+	int len = core->blocksize;
+	int tbs = len;
+	len = (int)rz_num_get(core->num, argv[1]);
+	if (len > tbs) {
+		rz_core_block_size(core, len);
+	}
+
+	switch (state->mode) {
+	case RZ_OUTPUT_MODE_JSON:
+		core_analysis_bytes_json(core, core->block, len, 0, state->d.pj);
+		break;
+	case RZ_OUTPUT_MODE_STANDARD:
+		core_analysis_bytes_standard(core, core->block, len, 0);
+		break;
+	default:
+		rz_warn_if_reached();
+	}
+
+	if (tbs != core->blocksize) {
+		rz_core_block_size(core, tbs);
+	}
```

Comment:
Avoid using `int`, especially for sizes, since they cannot be negative (in this case `ut32` should be ok). also throw errors for invalid values, like `zero` in this case.